WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -0,0 +1,36 @@
+/// FeatureFlag exposes a series of features to be conditionally enabled on
+/// different builds.
+@objc
+enum FeatureFlag: Int {
+    case exampleFeature
+    case newMediaExports
+    case pluginManagement
+    case googleLogin
+    case jetpackDisconnect
+
+    /// Returns a boolean indicating if the feature is enabled
+    var enabled: Bool {
+        switch self {
+        case .exampleFeature:
+            return true
+        case .newMediaExports:
+            return Build.is([.localDeveloper, .a8cBranchTest])
+        case .pluginManagement:
+            return Build.is(.localDeveloper)
+        case .googleLogin:
+            return Build.is(.localDeveloper)
+        case .jetpackDisconnect:
+            return Build.is(.localDeveloper)
+        }
+    }
+}
+
+/// Objective-C bridge for FeatureFlag.
+///
+/// Since we can't expose properties on Swift enums we use a class instead
+class Feature: NSObject {
```

Comment:
I don't see any uses of this in ObjC. Do we have to expose this to ObjC?

Reply:
I see at least three:

![screen shot 2017-10-04 at 09 19 21](https://user-images.githubusercontent.com/8739/31163946-34873f38-a8e5-11e7-96e1-39fe400f1728.png)

Reply:
Well damn, I completely botched my search.